package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -131,15 +129,10 @@
         ...
     ]
 
-Using the :meth:`WaterBridgeAnalysis.generate_table` method one can reformat
```

Comment:
Why did you remove the docs? Can they be added back elsewhere?

Reply:
@orbeckst Sorry, I initially removed it since this not class is not inherited from hydrogen bond analysis so I removed it. It has now been added back so this should be added back as well.